lib/Doctrine/DBAL/Platforms/AbstractPlatform.php

```diff
@@ -2837,7 +2837,7 @@ public function getListUsersSQL()
     /**
      * Returns the SQL to list all views of a database or user.
      *
-     * @param string $database
+     * @param string|null $database
```

Comment:
Why null?

Reply:
For the same reason, see an example among many: https://github.com/doctrine/dbal/blob/6c789b30a46906c5e669158809076d2405ea4fa9/lib/Doctrine/DBAL/Platforms/DB2Platform.php#L337-L340